bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/First.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkiverse.operatorsdk.bundle.sources;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.Version;
+
+@Group("samples.javaoperatorsdk.io")
+@Version("v1alpha1")
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Why is this needed?

Reply:
It's not needed. It's there because I copied it from the ping-pong example.